docs/porting/overview.md

```diff
@@ -1,5 +1,68 @@
-# 🔧 Overview of porting process
+# Overview of the Porting Process
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
\ No newline at end of file
+If you're coming from .NET Framework you may be interested in running your code on .NET Core.  Before any process is outlined, it's helpful to take a step back and evaluate your reasons for wanting to port code to run on .NET Core.
+
+## Should you port?
+
+The first thing you need to do is evaluate if it makes sense to port .NET Framework code over to .NET Core in the first place.
+
+Your code could be a good candidate for porting to .NET Core if...
+
+- Your code is a library and you want to expand its reach
+- You have a business need to run code on more than just Windows
+- You have expertise on non-Windows platforms, such as great Linux sysadmins
+- You're looking for an opportunity to implement an architectural change for your services (e.g. a monolithic service to microservices deployed to containers) or otherwise improve your codebase
+- You love new technology and want to cut your teeth on running code cross-platform
+
+Your code may not be a good candidate for porting if...
+
+- Your code is heavily tied to a Windows GUI or depends entirely on Windows-specific features
+- You have no business needs or expertise which would warrant running on more than Windows
```

Comment:
I think .NET Core offers more than cross-platform. It also offers app-local deployment and self-contained console apps.